libcodechecker/server/db_cleanup.py

```diff
@@ -0,0 +1,34 @@
+import sqlalchemy
+
+import shared
+from codeCheckerDBAccess_v6 import constants, ttypes
+from codeCheckerDBAccess_v6.ttypes import*
+
+from libcodechecker.logger import LoggerFactory
+from libcodechecker.server.database.run_db_model import *
+
+LOG = LoggerFactory.get_new_logger('DB CLEANUP')
+
+
+def remove_unused_files(session):
+    LOG.info("Garbage collection of dangling files started...")
+
+    bpe_files = session.query(BugPathEvent.file_id).subquery()
+    brp_files = session.query(BugReportPoint.file_id).subquery()
+
+    session.query(File) \
```

Comment:
FileContents already cleaned up? Don't we need some cleanup code for them as well?